util/cron/test-slurm-gasnet-ibv.numa.bash

```diff
@@ -1,14 +1,13 @@
 #!/usr/bin/env bash
 #
 # Multi-node, multi-locale testing on a linux64 cluster with slurm-gasnetrun_ibv launcher:
-# test numa locale model with comm=gasnet on multilocale and examples.
+# test numa locale model with comm=gasnet against "examples"
 
 CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
 source $CWD/common-slurm-gasnet-ibv.bash
-source $CWD/common-numa.bash
 
-export CHPL_NIGHTLY_TEST_CONFIG_NAME="slurm-gasnet-ibv.numa"
+export CHPL_LOCALE_MODEL=numa
 
-export GASNET_QUIET=Y
```

Comment:
That's likely because the need for it is due to something other than the platform.  AFAIK we've only been bothered by excess output that GASNET_QUIET removes when we use the udp or mpi conduits on systems that have better (notably Cray X* or anything with an InfiniBand network).  I suspect we would (do) need it when we test with conduit=udp on chapcs*, for example, or GASNet would produce output saying we weren't using the best conduit for the target.